README.md

```diff
@@ -78,18 +75,21 @@ For more advanced scenarios, please check the [terraform-dcos documentation for
 
 **NOTE:** This `quickstart` will provision a Kubernetes cluster without `RBAC` support.
 
-To deploy a cluster with enabled [RBAC](https://docs.mesosphere.com/services/kubernetes/1.2.2-1.10.7/authn-and-authz/#rbac) update `.deploy/options.json`:
+To deploy a cluster with enabled [RBAC](https://docs.mesosphere.com/services/kubernetes/2.0.0-1.12.1/authn-and-authz/#rbac) update `.deploy/options.json`:
 
 ```
 {
+  "service": {
+    "name": "dev/kubernetes01"
+  },
   "kubernetes": {
     "authorization_mode": "RBAC",
     "public_node_count": 1
   }
 }
 ```
 
-If you want to give users access to the Kubernetes API check [documentation](https://docs.mesosphere.com/services/kubernetes/1.2.2-1.10.7/authn-and-authz/#giving-users-access-to-the-kubernetes-api).
+If you want to give users access to the Kubernetes API check [documentation](https://docs.mesosphere.com/services/kubernetes/2.0.0-1.12.1/authn-and-authz/#giving-users-access-to-the-kubernetes-api).
```

Comment:
This link is broken 🤔 . https://docs.mesosphere.com/services/kubernetes/2.0.0-1.12.1/operations/authn-and-authz/#giving-users-access-to-the-kubernetes-api